### DISCUSSION TITLE
What's the company attitude towards Gatsby

### DISCUSSION THREAD

**Discussion by uro** [medusajs/medusa#3712] [Q&A] [answered]
I noticed that the Gatsby starter/source hadn't been updated in a long time. 

What is the company's strategy towards it?
I noticed that your admin panel was rewritten from Gatsby to different technology. 
Is next.js performing visibly better, so it will be better to migrate from Gatsby?

Do you plan to update the Gatsby source/starter? I didn't use it yet, so I don't know if it's missing something specific, but on the other hand, you're doing plenty of changes and next.js libraries are updated frequently.

Thanks for the answers!

**Comment by olivermrbl** [comment, marked as answer]
@uro, Our core team no longer maintains the Gatsby starter. We've moved all our frontend framework starter efforts to Next.js to avoid having too many active projects requiring our attention. 

Regarding the Gatsby source plugin, the story is somewhat the same. It has not been updated for a while, and we don't have any near-term plans of picking up that work. 

The most reasonable thing would probably be to deprecate the package in an upcoming release. I will bring this along to the team.

**Reply by uro** [reply]
Thanks for clearing that out. I will look towards migrating my project from Gatsby to Next.js or fitting native Medusa SDK into an existing project.
